promql/testdata/functions.test

```diff
@@ -514,3 +514,11 @@ eval instant at 0m days_in_month(vector(1454284800))
 eval instant at 0m days_in_month(vector(1485907200))
   {} 28
 
+clear
+
+# Test duplicate labelset in promql output.
+load 5m
+  testmetric1{src="a",dst="b"} 0
```

Comment:
No labelset does not contain `__name__`.
Acc to https://www.robustperception.io/whats-in-a-__name__

> If I multiply process_cpu_seconds_total by 0 or take a rate(), the resultant metric is no longer the amount of CPU time in seconds that a process has used since startup - so process_cpu_seconds_total no longer makes sense as a metric name. PromQL doesn't know what name would make sense, so the metric name (contained in the __name__ label) is removed.